pwiz_tools/Skyline/Util/BioMassCalc.cs

```diff
@@ -426,6 +426,15 @@ public BioMassCalc(MassType type)
             AddMass(Mn, 54.938045);
             AddMass(Mg, 24.305);
             AddMass(Si, 28.085); // Per Wikipedia
+            // Add all other elements
+            foreach (var entry in IsotopeAbundances.Default)

```

Comment:
Can you add documentation of where the values in IsotopeAbundances come from?

Reply:
Can we make this a TODO and accept this merge? Another request for it today.